tests/extension/oneapi_device_global/device_global_functional_several_kernels_one_device.cpp

```diff
@@ -0,0 +1,154 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides functional tests for device_global
+//
+//  Run several kernels on one device. The test modify device_global value in
+//  the first_kernel and then expect to read the same value from the second
+//  kernel. The test have a result array, that kernel uses through a buffer
+//  accessor. For the result array defined enum class is defined that have to be
+//  used for addressing elements in the array.
+//
+*******************************************************************************/
+
+#include "../../common/common.h"
+#include "../../common/type_coverage.h"
+#include "../../util/array.h"
+#include "device_global_common.h"
+#include "type_pack.h"
+
+#define TEST_NAME device_global_functional_several_kernel_in_one_device
+
+namespace TEST_NAMESPACE {
+using namespace sycl_cts;
+using namespace device_global_common_functions;
+#if defined(SYCL_EXT_ONEAPI_PROPERTY_LIST) && \
+    defined(SYCL_EXT_ONEAPI_DEVICE_GLOBAL)
+namespace oneapi = sycl::ext::oneapi;
+
+namespace several_kernel_in_one_device {
+template <typename T>
+oneapi::device_global<T> dev_global;
+template <typename T>
+oneapi::device_global<T> const_dev_global;
```

Comment:
Should we have
```suggestion
oneapi::device_global<std::add_const_t<T>> const_dev_global;
```
?
I do not understand well what you are trying to inthis test.

Reply:
I forgot to put the `const` modifier to the variable. It should be  `const oneapi::device_global<T> const_dev_global` as in other tests. I fixed it in 3f512947af398d2ca0aa730c7c3b11276e411931.

In this test, I'm testing a case, when a user tries to access device_global instance from different kernels on one device. In the first kernel, the instance is being modified and in the second kernel, we expect, that the instance will contain the changed value. With `const_dev_global` we expect to get the default value from the second kernel.